classes/log.php

```diff
@@ -10,7 +10,7 @@ public static function setOpensocialUserId($opensocial_user_id) {
 		self::$opensocial_user_id = $opensocial_user_id;
 	}
 
-	public static function _init(){
+	public static function _init() {
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed
* Method name "_init" should not be prefixed with an underscore to indicate visibility
* Opening brace should be on a new line